CHANGELOG.md

```diff
@@ -1,3 +1,160 @@
+# 85.2.0
+
+In this release we have added new features to Banner and Tabs, fixed bugs in Buttons and FlexibleHeader, and improved test coverage in BottomSheet. Additionally, we have added a TextField extension that will eventually house an MDCTextField alternative.
+
+## New features
+
+MDCBanner now supports `intrinsicContentSize`.
+
+```objc
+MDCBannerView *myBanner = ...
+CGSize validIntrinsicContentSize = myBanner.intrinsicContentSize;
+```
+
+MDCTabBarView has several new core features implemented.
```

Comment:
What does core feature mean here?

Reply:
Things that you'd reasonably expect in any v1 thing but that weren't there before

Reply:
@romoore suggests just not mentioning it, is that alright? 🤫

Maybe same with MDCBaseTextField?

Reply:
Took out any mention of MDCTabBarView and MDCBaseTextField